project/packaging.scala

```diff
@@ -84,6 +84,8 @@ object Packaging {
   maven-central
   sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots
   sonatype-releases: https://oss.sonatype.org/content/repositories/releases
+  jcenter-mvn: http://jcenter.bintray.com/
+  jcenter-ivy: http://jcenter.bintray.com/, [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier]).[ext]
```

Comment:
Please double check the syntax of these resolvers; I believe both jcenter-* resolvers are needed, as there might be both maven or ivy artifacts published to it.